python/ml4ir/base/data/ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--inputfilePath', type=str, default='', help='ranklib input file path')
+parser.add_argument('--outputfilePath', type=str, default='', help='output converted file path')
+parser.add_argument('--keepAdditionalInfo', type=int, default=1, help='Option to keep additional info (All info after the "#") 1 to keep, 0 to ignore')
```

Comment:
You may want to check [this answer for booleans](https://stackoverflow.com/a/15008806/1881878). Essentially here we want a bool and using an int is a hack!

Reply:
Same for `GL2Clicks` below!